_templates/content-source/new/prompt.js

```diff
@@ -0,0 +1,7 @@
+module.exports = [
+  {
+    type: 'input',
+    name: 'block_name',
+    message: 'What is the content source block called? Type `collection` for Collection Source Block.',
```

Comment:
would this be Collection Content Source Block? looking at the package.json
```suggestion
    message: 'What is the content source block called? Type `collection` for Collection Content Source Block.',
```